### ISSUE TITLE
"Surviving" notation backwards?

### ISSUE THREAD

**Issue by garyForeman** [boxed/mutmut#21]
Hello. First let me say I'm super excited for this repo! I literally learned about mutation testing today and immediately pip installed your package.

What's confusing me is what you call a "surviving mutant." My expectation was that surviving mutants would be those that still pass the test suite. However, [line 388](https://github.com/boxed/mutmut/blob/c86bde07edaa5340c57b4bb639e7c2f6b7c80a46/mutmut/__main__.py#L388) defines surviving mutants as those that do *not* pass the test suite. This means mutations that aren't a problem wind up being reported as "FAILED," and I haven't found a way to show the mutations that have left the test in a passing state.

TL;DR I want to know which mutants passed my test suite, not which ones don't pass my test suite.

My proposal is to remove `not` from line 388, but this drastically changes the behavior of this code base. Like I said, I'm a mutation testing noob, so it's entirely possible I'm thinking about this the wrong way.

Thanks again for the awesome repo!

**Comment by boxed** [comment]
You were absolutely correct. I screwed up!

I’ve pushed the change you suggested after having written a much more comprehensive test that will find this bug. Thanks for reporting this!

**Comment by garyForeman** [comment]
Great! Glad I was helpful, and thanks again for the nice repo!

**Comment by boxed** [comment]
Overall mutmut has had pretty weak tests for most of the time which is pretty ironic :P

I have toyed with the idea of running mutmut on mutmut but well... time is an issue obviously.
